frontend-react/src/network/QueryClients.ts

```diff
@@ -12,7 +12,12 @@ export const appQueryClient = new QueryClient({
         },
     },
 });
-export const testQueryClient = new QueryClient({
-    // to allow for faster testable failures
-    defaultOptions: { queries: { retry: false } },
-});
+export const getTestQueryClient = () =>
+    new QueryClient({
+        // to allow for faster testable failures
+        defaultOptions: {
```

Comment:
Since our approach is moving toward instantiating a new QueryClient for each test case, should we use [the same configuration as in production](https://github.com/CDCgov/prime-reportstream/pull/8349/files#diff-aa5131257f5b08b05099cac4d6c0fa3c367c4f16059043512369f72d668d6494R6-R11) so the behavior is the same?

Reply:
I ran into issues where turning on the suspend feature created errors when testing hooks in isolation.